RetrospectiveExtension.Frontend/components/feedbackItem.tsx

```diff
@@ -1,5 +1,5 @@
 import classNames from 'classnames';
-import * as moment from 'moment';
```

Comment:
since moment() is a function it can't be imported as a namespace and has to be imported as a function. see [here](https://www.typescriptlang.org/tsconfig#allowSyntheticDefaultImports) for more details